src/v2/Apps/Consign/Routes/MarketingLanding/Components/Header.tsx

```diff
@@ -1,83 +1,67 @@
 import React from "react"
-import styled from "styled-components"
-import { Media } from "v2/Utils/Responsive"
 import { SectionContainer } from "./SectionContainer"
 import { RouterLink } from "v2/System/Router/RouterLink"
-
-import {
-  Box,
-  Button,
-  Flex,
-  ResponsiveImage,
-  Sans,
-  Spacer,
-  Image,
-} from "@artsy/palette"
+import { FullBleedHeader } from "v2/Components/FullBleedHeader"
+import { Box, Button, Flex, Sans, Spacer } from "@artsy/palette"
+import { Media } from "v2/Utils/Responsive"
 
 export const Header: React.FC = () => {
   return (
-    <SectionContainer background="black5" height={[415, 546]} constrain={false}>
-      <HeaderImageContainer>
-        <Flex
-          width="100%"
-          justifyContent="space-between"
-          m="auto"
-          maxWidth={1280}
-        >
-          <Media greaterThan="xs">
-            <Image
-              src="https://files.artsy.net/consign/header-1.jpg"
-              width={380}
-              height={550}
-            />
-          </Media>
-          <CenterImage>
-            <ResponsiveImage src="https://files.artsy.net/consign/header-2.jpg" />
-          </CenterImage>
-          <Media greaterThan="xs">
-            <Image
-              src="https://files.artsy.net/consign/header-3.jpg"
-              width={380}
-              height={550}
-            />
-          </Media>
+    <SectionContainer height={[320, 546]} constrain={false} position="relative">
+      <Box position="relative">
+        <Flex style={{ objectFit: "cover" }}>
+          <FullBleedHeader src="http://files.artsy.net/consign/swa_landing_header.jpg" />
```

Comment:
If you like: this header component accepts children, which are positioned relative to the wrapper.

Reply:
(The intent there is to support overlays, etc.)